.circleci/config.yml

```diff
@@ -174,6 +174,15 @@ jobs:
     steps:
       - checkout
 
+      - restore_cache:
+          key: bundle-v1-{{ checksum "ios/Gemfile.lock" }}-{{ arch }}
+
+      - restore_cache:
+          key: yarn-v1-{{ checksum "yarn.lock" }}-{{ arch }}
```

Comment:
Can you share cache with other tasks, like `lint-test`?
https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/1029/files#diff-1d37e48f9ceff6d8030570cd36286a61R18

Reply:
no, cache is shared within one job